exe/dirtree

```diff
@@ -47,6 +55,17 @@ template = File.read(File.expand_path(template_file))
 tree = root.as_json
 result = ERB.new(template).result binding
 
+if options[:local_dependencies]
+  scripts = result.scan(/(<script src="(.+)"><\/script>)/)
+  scripts_url = scripts.collect {|res| res[1]}
+  scripts_url.each do |url|
+    injection = '<script>'
+    injection<<open(url).read
+    injection<<'</script>'
+    result.sub!(/(<script src=".+"><\/script>)/) {|m| m=injection}
```

Comment:
use gsub for that with a block to replace each finding directly 
also separating this feature to a class would be optimal